src/renderer/components/screens/NoAccountSelectedScreen/styles.module.scss

```diff
@@ -0,0 +1,24 @@
+.NoAccountSelectedScreen {
+  flex-grow: 1;
+  .Background {
```

Comment:
`.Background` is not a modifier of `NoAccountSelectedScreen` but its own component, you can declare the style on root level, no need to nest it here